doc/lsst.cp.verify/using-cp-verify.rst

```diff
@@ -0,0 +1,81 @@
+.. py:currentmodule:: lsst.cp.verify
+
+#####################################
+Verifying calibrations with cp_verify
+#####################################
+
+`cp_verify` is designed to make the verification of calibrations against the criteria listed in `DMTN-101 <https://dmtn-101.lsst.io>`_ as simple as possible.  However, this has resulted in the tasks being rather abstract, with no clear connection between steps.  This document should help explain how the tests are run, how those results are stored, and how they are passed to `analysis_tools` for conversion into metrics and plots.
```

Comment:
I know you don't want a full review of formatting and everything, but the standard is that each sentence gets its own line in the rst doc files.  This is important to do before the first merge so that subsequent changes are easier to read in the github diff.

Reply:
Done here so the actual documentation ticket will have the cleaner diffs.